File: algorithms_and_data_structures/lecture-1/available_devider.java
import java.util.ArrayList;
import java.util.List;

public class available_devider {
    public static void main(String[] args){
        System.out.println(availableDevider(15));
    }
    public static List<Integer> availableDevider(int number) {
        List<Integer> result = new ArrayList<>();
        for (int i = 1; i <= number; i++) {
            if (number % i == 0) {
                result.add(i);
            }
        }
        return result;
    }
}

File: algorithms_and_data_structures/lecture-1/available_devider.py
# Программа для определения допустимых делителей для числа

def available_devider():
    number = int(input("Введите число: "))
    result = []
    for i in range(1, number+1):
        if number % i == 0:
            result.append(i)

    return result


print(available_devider())
